frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -141,6 +159,13 @@ const GitSection: React.FC<GitSectionProps> = ({ showSample, builderImages }) =>
     values.name,
   ]);
 
+  React.useEffect(() => {
+    values.build.strategy === BuildStrategyType.Source &&
+      !(values.image.couldNotRecommend || values.image.recommended) &&
+      handleBuilderImageRecommendation();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [handleBuilderImageRecommendation, values.build.strategy, builderImages]);
```

Comment:
Why not add all deps, this seems wrong to not include `values.image.couldNotRecommend` and `values.image.recommended`
For ever disabled eslint rule, there must be a comment describing exactly why the rule is disabled.